ask-sdk-lambda-support/src/com/amazon/ask/SkillStreamHandler.java

```diff
@@ -80,8 +81,8 @@ public SkillStreamHandler(final AlexaSkill... skills) {
     public final void handleRequest(final InputStream input, final OutputStream output, final Context context)
             throws IOException {
         byte[] inputBytes = IOUtils.toByteArray(input);
-        for (AlexaSkill skill : skills) {
-            SkillResponse response = skill.execute(new BaseSkillRequest(inputBytes), context);
+        for (int i = 0; i < skills.size(); ++i) {
```

Comment:
Nit: why not iterate over using foreach?

Reply:
Ha, my bad, changes while testing